src/sys/mman.rs

```diff
@@ -513,6 +513,7 @@ pub unsafe fn munmap(addr: NonNull<c_void>, len: size_t) -> Result<()> {
 /// [`MmapAdvise::MADV_FREE`].
 ///
 /// [`madvise(2)`]: https://man7.org/linux/man-pages/man2/madvise.2.html
+#[allow(rustdoc::broken_intra_doc_links)]
```

Comment:
Might be better to add a comment:

```suggestion
#[allow(rustdoc::broken_intra_doc_links)] // For Hurd as `MADV_FREE` is not available on it
```

Though AIX does not have it either, but we don't have CI for AIX, so no tests, no warnings :>

Reply:
ok, done so